conflux/test/adapter.test.ts

```diff
@@ -0,0 +1,99 @@
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
+// import { Config } from '../src/config'
+
+const TestOracleAddress = '0x8688ebA9Bf38cBb1Fa27A8C3Fda11414D6057887'
+const FunctionSelector = '0x4ab0d190'
+const DataPrefixExample =
+  '0x9b8c5ab8dc03bb19d6a5047c213537c5000d151e66e49458e054c68cb504904f0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000869795b26b0f5f00ba2b3cf3380634fb5f4806094357855e000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000060346c9e'
+
+describe('execute', () => {
+  const jobID = 'fd26a90e0aa84040bc6b4d6f5036a23a'
+  const execute = makeExecute()
```

Comment:
Config is useful to configure the adapter, and we could configure it when we create it (`makeExecute`). You could simply configure it as:
```js
const execute = makeExecute({ rpcUrl: '', privateKey: '', networkId: 1, api: {} })
```
This will skip your default configuration, that requires some ENV vars. 

Even though we cannot test the integration as it needs to make real tx's into Conflux, we still can make the validation tests.

Reply:
Remove the noisy comments in the whole adapter as well please